tools/hangar/frontend/src/components/tabs/OsqueryPerfTab.tsx

```
@@ -2072,7 +2083,7 @@ function SmallField({
   );
 }
 
-/// Builds the argv tail (after `agent.go`). Kept here so the preview
+/// Builds the argv tail (after `go run .`). Kept here so the preview
```

ESLint

[error] Expected exception block, space or tab after '//' in comment.

(spaced-comment)